<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Analog_Viewer</class>
 <widget class="QWidget" name="Analog_Viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="JKQTPlotter" name="plot" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>561</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>561</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Graph Editor</string>
   </property>
   <widget class="QComboBox" name="graphchoose_cbox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Graph of Interest:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="yheight_dspinbox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>99999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y height:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>58</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y offset:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="yoffset_dspinbox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>99999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QCheckBox" name="show_checkbox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_pushbtn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>55</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showaxis_checkbox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Axis</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X axis width:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="xwidth_dspinbox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>501</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="minimum">
    <double>0.000010000000000</double>
   </property>
   <property name="maximum">
    <double>100000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>50.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="snapto_pushbtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Snap playhead to center</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>JKQTPlotter</class>
   <extends>QWidget</extends>
   <header>jkqtplotter/jkqtplotter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
